packages/contracts/contracts/core/plugin/ProposalBase.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {IDAO} from "../IDAO.sol";
+
+/// @title ProposalBase
+/// @author Aragon Association - 2022
+/// @notice An abstract base xcontract defining the traits and internal functionality to create and execute proposals.
+abstract contract ProposalBase {
+    /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
+    bytes4 internal constant PROPOSAL_INTERFACE_ID = this.proposalCount.selector;
```

Comment:
shouldn't we also include `createProposalId()` ?

Reply:
Not possible because `createProposalId()` is an internal function.
When the `proposalId` as `bytes32` PR will be merged in, the interface will become more "meaningful"